src/pages/MovieDetailsPage.vue

```diff
@@ -1,20 +1,94 @@
 <template>
-  <div>
-    <h1>Movie Details</h1>
-    <h3>Movie ID: {{ movieId }}</h3>
+  <div class="movie-page">
+    <div class="movie-page__description">
+      <h1>{{ movie[0].title }}</h1>
+      <div>
+        <span class="movie-page__description-genre">{{
+          movie[0].genre.name
+        }}</span>
+        <span class="movie-page__description-length">{{
+          toHoursAndMinutes(movie[0].length)
+        }}</span>
+      </div>
+      <p class="movie-page__description-text">{{ movie[0].description }}</p>
+    </div>
+    <div class="movie-page__image">
+      <img :src="movie[0].poster_url" alt="" />
+    </div>
   </div>
 </template>
 
 <script>
 export default {
   name: "MovieDetailsPage",
+  data() {
+    return {
+      movie: this.$store.getters.movieList.filter(
+        (movie) => movie.id == this.movieId
+      ),
```

Comment:
mayby `.find` would be a better fit? 🤔

Reply:
It is definatelly much more fitting, thank you 👍🏽